addons/binding/org.openhab.binding.squeezebox/src/main/java/org/openhab/binding/squeezebox/internal/utils/HttpUtils.java

```diff
@@ -56,14 +58,20 @@ public static String post(String url, String postData) throws Exception {
         ContentResponse response = client.newRequest(url)
                 .method(HttpMethod.POST)
                 .content(new StringContentProvider(postData))
-                .timeout(TIMEOUT, TimeUnit.MILLISECONDS)
+                .timeout(timeout, TimeUnit.MILLISECONDS)
                 .send();
 
         int statusCode = response.getStatus();
 
+        if (statusCode == HttpStatus.UNAUTHORIZED_401) {
+            String statusLine = response.getStatus() + " " + response.getReason();
+            logger.error("Received '{}' from squeeze server", statusLine);
+            throw new NotAuthorizedException("Unauthorized: " + statusLine);
```

Comment:
> It's not black-and-white

Yea, I know.  But, before I used it, I searched the repos for any use of that exception, and there was none.  So I figured it might be questionable.